docs/Configuration/yaml-files.md

```diff
@@ -351,6 +353,7 @@ install_script:
 uninstall_script:
   path: ../lib/software/tailscale-uninstall-script.ps1
 self_service: true
+auto-install: true
```

Comment:
Dev note:

1. If the admin has an existing policy installing the same app, show an error: "Only one version of an app can be available for install (aka added) per team."